core/src/main/resources/application.yml

```diff
@@ -323,6 +323,8 @@ cb:
     user.role: "ROLE_USER"
     heartbeat.interval: 3
     missed.heartbeat.interval: 30
+    sharedComponents: >
+      HDFS=JOURNALNODE
```

Comment:
Just a question butmaybe we should use a config json ?

Reply:
I think it would add unnecessary complexity. Also it would require a rebuild to run with a new config. On the other hand this property can be modified without building a new CB version.

Reply:
which is easier to update on the deployed version? in app.yml or in a json?

Reply:
I think it is the same. I am just saying that somehow it is not clear what is the key and the value here ?

Reply:
json is built into the artifact. app.yml properties can be overwritten with env vars.

Reply:
in cb deployer we have capabilities to override any json which contains config. It is true that we can not do the same on the hosted service but the question here is that do we really need such flexibility ? :)

Reply:
If we do not want to change on the hosted service, in this case, we do not need to this flexibility.